kili/cli/project/member/add.py

```diff
@@ -0,0 +1,197 @@
+"""CLI's project member add subcommand"""
+
+import csv
+import re
+from typing import Dict, List, Optional, cast
+import warnings
+import click
+
+from kili.client import Kili
+from kili.cli.common_args import Options
+
+
+REGEX_EMAIL = re.compile(
+    r'([A-Za-z0-9]+[.-_])*[A-Za-z0-9]+@[A-Za-z0-9-]+(\.[A-Z|a-z]{2,})+')
+ROLES = ["ADMIN", "TEAM_MANAGER", "REVIEWER", "LABELER"]
+
+
+def collect_members_from_csv(csv_path: str, role: Optional[str]):
```

Comment:
I have quite a big comment on the collect_members_from_csv, I will call you to discuss this with you